src/index.php

```diff
@@ -1,6 +1,9 @@
 <?php
 
-require_once __DIR__ . '/../vendor/autoload.php';
+// Define application root directory
+define('APP_DIR', dirname(__DIR__));
+
+require_once APP_DIR . '/vendor/autoload.php';
```

Comment:
@jetbrains-junie don't include this file, this project doesn't use composer. Composer is just used during testing.